Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanel.swift

```diff
@@ -73,7 +73,12 @@ struct FloatingPanel<Content>: View where Content: View {
             }
             // Set frame width to infinity to prevent horizontal shrink on dismissal.
             .frame(maxWidth: .infinity)
+#if os(visionOS)
+            .background(.regularMaterial)
+            .compositingGroup()
```

Comment:
Without using the `compositingGroup()` modifier you end up with something like this:
![Material_Showing](https://github.com/user-attachments/assets/18d789be-18ed-485f-974d-78ef9843daf0)

I believe adding this modifier lets the material layer get added **before** the clipping occurs.